ui/test/forge_shell_test.cpp

```
@@ -104,8 +111,10 @@ class TestDocumentHost final : public DocumentHost {
   void markClean() noexcept { savedRecords_ = doc_.records().size(); }
   std::size_t saves() const noexcept { return saves_; }
   std::size_t opens() const noexcept { return opens_; }
+  std::size_t resets() const noexcept { return resets_; }
```

Cppcheck (2.21.0)

[style] The function 'resets' is never used.

(unusedFunction)